src/BigInteger.php

```diff
@@ -386,6 +386,28 @@ public function gcd($that) : BigInteger
         return new BigInteger($value);
     }
 
+    /**
+     * Returns the square root number of this number and the given one.
+     *
+     * @param BigNumber|number|string $that The number to add. Must be convertible to a BigInteger.
+     *
+     * @return BigInteger The result.
+     *
+     * @throws MathException If the number is not valid, or is not convertible to a BigInteger.
+     */
+    public function sqrt($that) : BigInteger
+    {
+        $that = BigInteger::of($that);
```

Comment:
Actually no need to check the operand here, as we're already in a `BigInteger`, so `$this->value` is already validated.